strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/MavenMetadataServiceHelper.java

```diff
@@ -0,0 +1,42 @@
+package org.carlspring.strongbox.testing;
+
+import org.carlspring.strongbox.artifact.locator.ArtifactDirectoryLocator;
+import org.carlspring.strongbox.event.artifact.ArtifactEventListenerRegistry;
+import org.carlspring.strongbox.locator.handlers.GenerateMavenMetadataOperation;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.providers.io.RepositoryPathResolver;
+import org.carlspring.strongbox.storage.metadata.MavenMetadataManager;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import javax.inject.Inject;
+import java.io.IOException;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Pablo Tirado
+ */
+@Component
```

Comment:
@ptirador you don't need a `@Component` annotation here as it defined within `IntegrationTestsConfiguration` configuration class.